benchmark/benchmark.go

```diff
@@ -222,12 +226,25 @@ func NewClientConn(addr string, opts ...grpc.DialOption) *grpc.ClientConn {
 	return conn
 }
 
-func runUnary(b *testing.B, maxConcurrentCalls, reqSize, respSize int) {
+func runUnary(b *testing.B, maxConcurrentCalls, reqSize, respSize, kbps, mtu int, ltc time.Duration) {
 	s := stats.AddStats(b, 38)
+	nw := &latency.Network{Kbps: kbps, Latency: ltc, MTU: mtu}
+	nw.Kbps = -1
```

Comment:
What's the purpose of this?
This function should always set `nw.Kbps` to the parameter `kbps` passed in.

If we want to run benchmarks only with `kbps` -1, we should make sure this function is called with `kbps=-1`, instead of setting `nw.Kbps` to -1 inside the function.